src/main/default/lwc/voteTracker/voteTracker.js

```diff
@@ -0,0 +1,176 @@
+import { LightningElement, api } from 'lwc';
+import voteQuestion from '@salesforce/label/c.cookbook_VoteQuestion';
+import voteLabel from '@salesforce/label/c.cookbook_VoteLabel';
+import voteFinalText from '@salesforce/label/c.cookbook_VoteFinalText';
+import abandonRequestText from '@salesforce/label/c.cookbook_AbandonRequest';
+import yesLabel from '@salesforce/label/c.cookbook_Yes';
```

Comment:
The `Label` suffix is redundant since all the labels are wrapped in the `labels` object. I'd suggest to simply use `yes` and `no`.